src/modules/common/components/chevron-flip/chevron-flip.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+import Styles from 'modules/common/components/chevron-flip/chevron-flip.styles'
+
+const ChevronFlip = p => (
+  <svg
+    className={classNames(
+      Styles.ChevronFlip,
+      p.className,
+      { [Styles.pointDown]: p.pointDown },
+    )}
+    viewBox="0 0 16 16"
+  >
+    <g id="Symbols" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd" strokeLinecap="round" strokeLinejoin="round">
```

Comment:
Are the IDs necessary? If so, can they be classes?

Reply:
No, the ids are not necessary. It's just how the SVG is auto-exported from Sketch. I've removed them.